testing/unit-testing-jest/docs/test-structure.md

```diff
@@ -0,0 +1,19 @@
+# Basic structure of a test 
+````ts
+    🏷️ describe('Class, Module, Service or Component to test', () => {
+        //test
+       🌱️ it('should do ...', () => {
+            // setup
+            const service = new ServiceToTest();
+            // assertion and matcher
+           🔍️️ expect(service.fn)️.🧪toBe('expected value');
+        });
+    });
+````
+🏷  First use ```describe```to define the scope of the tests, this can contain one o more test, inside yo define the name of the class that you wanna test, next put a call back ```,() =>```
```

Comment:
```suggestion
🏷  First use ```describe```to define the scope of the tests, this can contain one o more test, inside define the name of the class that you wanna test, next put a call back ```,() =>```
```